logos/logos-ui/src/app/features/ai-tools/ai-tools.scss

```
@@ -10,46 +10,346 @@
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-semibold);
+
+  i { font-size: var(--font-size-sm); }
+}
+
+.stepper-title { white-space: nowrap; }
+
 // ── Section cards ─────────────────────────────────────────────────────────
 .config-section {
   @include glass-surface;
   margin-top: 16px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)